internal/services/withexamples.go

```diff
@@ -5,4 +5,5 @@ package services
 // Stop calls the brokerManager function that stops and cleans the examplebroker files.
 func (m *Manager) Stop() {
 	m.brokerManager.Stop()
+	return m.stop()
 }
```

Comment:
This change does not look necessary if we don't actually update the func return...

Reply:
you’re right about the error, fixing.